ftw/iframeblock/profiles/default/jsregistry.xml

```diff
@@ -1,9 +1,11 @@
 <?xml version="1.0"?>
 <object name="portal_javascripts" meta_type="JavaScripts Registry">
+
   <javascript enabled="True" id="++resource++ftw.iframeblock/iframeResizer.contentWindow.min.js"
```

Comment:
A thought in passing:
IIUC `iframeResizer.contentWindow.min.js` gets bundled in production which means:
a) it is useless for "contained sites" to reference
b) it is not used by this site either (unless we're contained within an iframe)

So I'm thinking we should either:
i) exclude the file, OR
ii) stop it being bundled (and move it to the end of jsregistry.xml to not disrupt other bundles) AND document it's use it in README :-) 
But probably a separate PR

Reply:
See comment above about why we need it https://github.com/4teamwork/ftw.iframeblock/pull/27#discussion_r388775981

We have to bundle it into the `compiled.js` because otherwise, it won't be present on Plone 5.

Reply:
Yeah ok. I was more focussing on Plone4.  

My thoughts started with the best way for the 'contained site' to get a _matching_ version of iframeResizer is to source it from our site.

Reply:
Ok leave this for now - raised as #28